cp3pt0-deployment/setup_singleton.sh

```diff
@@ -220,7 +243,7 @@ function is_migrate_licensing() {
     get_and_validate_arguments
     if [ ! -z "$CONTROL_NS" ]; then
         if [[ "$CUSTOMIZED_LICENSING_NAMESPACE" -eq 1 ]] && [[ "$CONTROL_NS" != "$LICENSING_NAMESPACE" ]]; then
-            error "Licensing Migration could only be done in $CONTROL_NS, please do not set parameter '-licensingNs $LICENSING_NAMESPACE'"
+            error "Licensing Migration could only be done in $CONTROL_NS, please do not set parameter '-lsNs $LICENSING_NAMESPACE'"
```

Comment:
see above comment - it seems we're changing the parameters name, which looks like a breaking  interface change

Reply:
reverted to -licensingNs